### ISSUE TITLE
error in rstudio when running ROGRSQL as an sql script

### ISSUE THREAD

**Issue by idshklein** [brownag/ROGRSQL#3]
when I try to run the example from the readme under RStudio Interface and RMarkdown, I get the following error:
![image](https://github.com/brownag/ROGRSQL/assets/12258810/39ec16c8-6970-40bf-87e7-cf03062eb216)

I use RStudio 2023.06.0+421 "Mountain Hydrangea" Release (583b465ecc45e60ee9de085148cd2f9741cc5214, 2023-06-05) for windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2023.06.0+421 Chrome/110.0.5481.208 Electron/23.3.0 Safari/537.36

**Comment by brownag** [comment]
Hi, thanks for reporting this. I was puzzled at first but have looped back with a workaround

This is a bug, or change in behavior, of RStudio. It probably should be reported over there, as it will occur even when using `RSQLite::SQLite()` (i.e. this problem is not unique to ROGRSQL)

These expressions used to work fine, and the RMarkdown analog seems to work. It seems the more complex expression used to get the sample dataset from the package is not being parsed correctly because the following works:

```sql
-- !preview conn=DBI::dbConnect(ROGRSQL::OGRSQL(), "/home/andrew/workspace/ROGRSQL/inst/extdata/lux.gpkg")

SELECT ST_Centroid(geom) FROM lux LIMIT 1;
```

but not 

```sql
-- !preview conn=DBI::dbConnect(ROGRSQL::OGRSQL(), system.file("extdata", "lux.gpkg", package="ROGRSQL"))

SELECT ST_Centroid(geom) FROM lux LIMIT 1;
```

So, I have added `ROGRSQL::sample_gpkg_path()` and used it in the README. 

```sql
-- !preview conn=DBI::dbConnect(ROGRSQL::OGRSQL(), ROGRSQL::sample_gpkg_path())

SELECT ST_Centroid(geom) FROM lux LIMIT 1;
```

In summary for you need to provide a static path _or_ simple R expression to avoid the parsing error. I doubt the change was intentional for RStudio. 

---

Unrelated to source of this issue, but related to rebuilding the README, I noticed that the dbplyr SQL translation is kinda broken. Not sure what changed or when there, will look into that soon

**Comment by brownag** [comment]
This bug has been reported to RStudio and replicated by them, so hopefully it will be fixed in the next release.

Closing this as the workaround for the readme/example data has been implemented via helper function`sample_gpkg_path()`

**Comment by idshklein** [comment]
amazing. thank you!
